apps/omg_watcher/test/integration/in_flight_exit_test.exs

```diff
@@ -107,6 +107,14 @@ defmodule OMG.Watcher.Integration.WatcherApiTest do
 
     Eth.DevHelpers.wait_for_root_chain_block(eth_height + exiters_finality_margin)
 
+    assert %{
```

Comment:
check if the "other" test case in this file doesn't have obsolete piggyback assertions now, which we can remove (or at least comment)